resources/definitions/hellbot_hidra_plus.def.json

```diff
@@ -0,0 +1,59 @@
+{
+    "version": 2,
+    "name": "Hellbot Hidra Plus",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Hellbot Development Team",
+        "manufacturer": "Hellbot",
+        "file_formats": "text/x-gcode",
+		"platform": "hellbot_hidra_plus.obj",
+		"platform_offset": [0, 0, 5],
+        "platform_texture": "hellbot_hidra_plus.png",
+        "has_materials": true,
+        "machine_extruder_trains":
+        {
+            "0": "hellbot_hidra_plus_extruder_0",
+			"1": "hellbot_hidra_plus_extruder_1"
+        }
+
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "Hellbot Hidra Plus" },
+        "machine_width": {
+            "default_value": 305
+        },
+        "machine_depth": {
+            "default_value": 305
```

Comment:
On the Hellbot website it claims a size of 300x300x350mm: https://hellbot.xyz/impresora3d-fdm-hidra/

Is the website incorrect here, or are these changes incorrect?